lib/resque/scheduler/delaying_extensions.rb

```diff
@@ -62,44 +62,34 @@ def enqueue_in_with_queue(queue, number_of_seconds_from_now,
       # if O(log(n)).  Returns true if it's the first job to be scheduled at
       # that time, else false
       def delayed_push(timestamp, item)
-        # First add this item to the list for this timestamp
-        redis.rpush("delayed:#{timestamp.to_i}", encode(item))
-
         # Store the timestamps at with this item occurs
         redis.sadd("timestamps:#{encode(item)}", "delayed:#{timestamp.to_i}")
```

Comment:
Yea, its so that the resque scheduler dashboard can show a mapping from timestamp -> job. FWIW if we decide to go shopify specific i think we can 🔥  a lot of this code.